packages/main/src/backend/export/outputVendors/ynab/ynab.test.ts

```diff
@@ -6,6 +6,42 @@ import * as ynab from './ynab';
 import ClearedEnum = SaveTransaction.ClearedEnum;
 
 describe('ynab', () => {
+  describe('isSameTransaction(Payee changed)', () => {
+    test('Two transactions with different payee names should be considered the same if they have the same import id', () => {
+      const transferTransactionFromYnab: TransactionDetail = {
```

Comment:
Rename as this is not a tranfer transaction
```suggestion
      const differentPayeeTransactionFromYnab: TransactionDetail = {
```

Reply:
Agreed, done.